src/crafting_gui.cpp

```diff
@@ -1693,6 +1706,24 @@ static bool query_is_yes( const std::string &query )
            subquery == _( "yes" );
 }
 
+static size_t craft_info_width( const size_t window_width )
```

Comment:
I suggest using signed integer here because this function is called with signed integer argument and the return value is also used as a signed integer.